ui/hooks/useNftsCollections.js

```diff
@@ -5,10 +5,10 @@ import { getNfts, getNftContracts } from '../ducks/metamask/metamask';
 import { getCurrentChainId, getSelectedAddress } from '../selectors';
 import { usePrevious } from './usePrevious';
 
-export function useNftsCollections() {
+export function useNftsCollections(previouslyOwnedText, unknownCollectionText) {
```

Comment:
Instead of passing these in, you can call `useI18nContext` within this hook (see here: https://github.com/MetaMask/metamask-extension/pull/18389)

Reply:
Done!